core/dnsserver/server_tls.go

```diff
@@ -87,3 +97,7 @@ func (s *ServerTLS) OnStartupComplete() {
 		fmt.Print(out)
 	}
 }
+
+const (
+	tlsMaxQueries = -1
```

Comment:
You are right I think I didn't mention it in the pull request, but did in the issue its linked to, so my bad. The default was never set originally by Coredns so is defaulted to 128 here: https://github.com/miekg/dns/blob/master/server.go#18

The change is to remove the limit since we are having timeouts for idle connections anyway and an active connection has no good reason to limit the amount of connections it can make. Especially given a malicious actor could just keep reconnecting and sending another 128 anyway.

I'm also sure I found a post on Github somewhere where miekg mentioned he wasn't sure the point of the limit especially with timeouts set hence why I hard-coded to -1 (no limit) rather than trying to make it configurable.

Reply:
> I think I didn't mention it in the pull request ...

You did mention it in the PR description, I just didn't see it before.  Thanks!

From the PR description...
> It also removes the limit of queries per connection since we are enforcing timeouts this makes no sense and can cause some devices to think the DNS server is faulty.